tests/rust_tests.rs

```diff
@@ -127,6 +137,43 @@ fn pause_test() {
         .assert_ok();
 }
 
+#[test] // Tests if the contract has whitelist enabled and is empty

```

Comment:
Should change comment to "Tests if the contract has whitelist enabled and is empty by default after deployment"